packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.types.ts

```diff
@@ -419,6 +429,11 @@ export interface IColumn {
    */
   onRender?: (item?: any, index?: number, column?: IColumn) => any;
 
+  /**
+   * If set, parent getCellValueKey will return this value.
+   */
+  getValueKey?: (item?: any, index?: number, column?: IColumn) => string;
```

Comment:
Are both `getCellValueKey` and `getValueKey` necessary? They seem redundant in signature and implementation in the example below.

Can we collapse it to only one new method to implement?

Reply:
Spoke offline, this is required for per column overrides similar to `IColumn#onRender`